cmd/dptp-controller-manager/main.go

```diff
@@ -174,6 +191,44 @@ func newOpts() (*options, error) {
 		opts.testImagesDistributorOptions.forbiddenRegistries.Insert(vals...)
 	}
 
+	opts.registrySyncerOptions.excludeImageStreamTags = sets.String{}
+	if vals := opts.registrySyncerOptions.excludeImageStreamTagsRaw.Strings(); len(vals) > 0 {
```

Comment:
We should probably factor the namespace/name and imagestreamtag flag parsing logic out